### DISCUSSION TITLE
Unable to transcribe video file

### DISCUSSION THREAD

**Discussion by kanishknishar** [openai/whisper#874] [Q&A]
This [video](https://we.tl/t-snBCvq7LWz) has 6 channel sound which normally whisper can transcribe but this time it failed.

```
Traceback (most recent call last):
  File "C:\Program Files\Python310\lib\site-packages\whisper\audio.py", line 42, in load_audio
    ffmpeg.input(file, threads=0)
  File "C:\Program Files\Python310\lib\site-packages\ffmpeg\_run.py", line 325, in run
    raise Error('ffmpeg', out, err)
ffmpeg._run.Error: ffmpeg error (see stderr output for detail)

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Program Files\Python310\Scripts\whisper-script.py", line 33, in <module>
    sys.exit(load_entry_point('openai-whisper==20230117', 'console_scripts', 'whisper')())
  File "C:\Program Files\Python310\lib\site-packages\whisper\transcribe.py", line 311, in cli
    result = transcribe(model, audio_path, temperature=temperature, **args)
  File "C:\Program Files\Python310\lib\site-packages\whisper\transcribe.py", line 85, in transcribe
    mel = log_mel_spectrogram(audio)
  File "C:\Program Files\Python310\lib\site-packages\whisper\audio.py", line 111, in log_mel_spectrogram
    audio = load_audio(audio)
  File "C:\Program Files\Python310\lib\site-packages\whisper\audio.py", line 47, in load_audio
    raise RuntimeError(f"Failed to load audio: {e.stderr.decode()}") from e
RuntimeError: Failed to load audio: ffmpeg version 5.1.2-essentials_build-www.gyan.dev Copyright (c) 2000-2022 the FFmpeg developers
  built with gcc 12.1.0 (Rev2, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-zlib --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-sdl2 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libgme --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libtheora --enable-libvo-amrwbenc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-librubberband
  libavutil      57. 28.100 / 57. 28.100
  libavcodec     59. 37.100 / 59. 37.100
  libavformat    59. 27.100 / 59. 27.100
  libavdevice    59.  7.100 / 59.  7.100
  libavfilter     8. 44.100 /  8. 44.100
  libswscale      6.  7.100 /  6.  7.100
  libswresample   4.  7.100 /  4.  7.100
  libpostproc    56.  6.100 / 56.  6.100
Input #0, mpegts, from '.\1 Intro.ts':
  Duration: 00:08:41.60, start: 10342.192578, bitrate: 22465 kb/s
  Program 1
  No Program
  Stream #0:0[0x134]: Video: h264 (High), yuv420p(tv, bt709, top first), 1920x1080 [SAR 1:1 DAR 16:9], 25 fps, 25 tbr, 90k tbn
Output #0, s16le, to 'pipe:':
Output file #0 does not contain any stream
```

How can I fix this issue?

**Comment by glangford** [comment]
You can use VLC (File / Convert-Stream) to extract the audio as .mp3 and `whisper` seems to accept that (ordinarily I would do any special conversion with `ffmpeg` directly but no mix of command line options seemed to work :/ )

https://www.videolan.org/vlc/

**Comment by glangford** [comment]
1
00:00:00,000 --> 00:00:07,000
Hello and welcome to The Young Vic. What an absolute pleasure it is to be broadcasting

2
00:00:08,460 --> 00:00:14,840
live from this world famous theatre here on London's Southbank. The Young Vic theatre

3
00:00:14,840 --> 00:00:19,600
really changes radically for each show that they produce. So you may not recognise it

4
00:00:19,600 --> 00:00:24,020
but we have broadcast from this space before, The Streetcar Named Desire, starring Gillian

5
00:00:24,020 --> 00:00:29,380
Anderson. And you may have seen The Young Vic's last NT live, A View from the Bridge,

6
00:00:29,380 --> 00:00:34,620
live from London's West End. It won major awards in the UK and went on to take Broadway

7
00:00:34,620 --> 00:00:41,200
by storm, winning two Tony Awards. This production of Yerba, originally produced in 2016, has

8
00:00:41,200 --> 00:00:46,920
been received just as enthusiastically, winning two Olivier Awards, an Evening Standard Award

9
00:00:46,920 --> 00:00:51,800
amongst many others. We'll tell you more about that thrilling reinvention of Federico

10
00:00:51,800 --> 00:00:57,800
Garcia Lorca's original play, In a Moment, in a short film featuring director Simon Stone

11
00:00:57,800 --> 00:01:03,520
and Young Vic artistic director David Lan. Now, as you'll have just seen, NT live is

12
00:01:03,520 --> 00:01:09,080
sponsored in the UK by Sky Arts. They support us in broadcasting world class theatre to

13
00:01:09,080 --> 00:01:14,000
cinemas and enable us to work with partners such as The Young Vic to bring the best theatre

14
00:01:14,000 --> 00:01:19,120
to our audiences. Right now we are coming to the end of a very busy summer but we've

15
00:01:19,120 --> 00:01:24,960
got some fantastic broadcast screenings over the autumn months still ahead. First, Hamlet,

16
00:01:24,960 --> 00:01:29,080
starring Benedict Cumberbatch, will be coming back to the cinemas, directed by Lindy Turner

17
00:01:29,080 --> 00:01:33,960
and produced by Sonia Friedman Productions. It was a smash when it was originally broadcast

18
00:01:33,960 --> 00:01:40,040
in 2015 and encore screenings will be in the UK and Ireland from the beginning of the 5th

19
00:01:40,040 --> 00:01:45,280
of October. And international screenings will begin from the 1st of March 2018, then on

20
00:01:45,280 --> 00:01:50,960
the 16th of November we'll be broadcasting Stephen Sondheim's legendary musical Follies,

21
00:01:50,960 --> 00:01:55,800
live from the National Theatre and directed by Dominic Cook. The multi-award winning musical

22
00:01:55,800 --> 00:02:02,040
will feature Tracey Bennet, Janie Dee and Imelda Staunton. With a cast of 37 and an

23
00:02:02,040 --> 00:02:07,080
orchestra of 21, it is the first time this epic musical will be staged at the National

24
00:02:07,080 --> 00:02:12,480
and it's not to be missed. On the 7th of December we will bring you the brand new broadcast

25
00:02:12,480 --> 00:02:17,840
production at the Bridge Theatre of Young Marks directed by Nicholas Hytner. And next

26
00:02:17,840 --> 00:02:22,960
spring we'll return to bring you Hytner's production of Julius Caesar. Finally, we're

27
00:02:22,960 --> 00:02:27,440
so excited to announce here first that we'll be collaborating with the Young Vic Theatre

28
00:02:27,440 --> 00:02:33,400
again to bring you Benedict Andrews' production of Tennessee Williams' Cat on a Hot Tin Roof.

29
00:02:33,400 --> 00:02:38,960
This special live captured event will hit cinemas in February 2019 and stars Jack O'Connell

30
00:02:38,960 --> 00:02:43,320
and Sienna Miller. Tickets are already on sale for many of these broadcasts so please

31
00:02:43,320 --> 00:02:47,520
do head to the website to find out more. You can also sign up for email updates to hear

32
00:02:47,520 --> 00:02:54,640
the first about future shows. Now, this production of Yerma is an adaptation written and directed

33
00:02:54,640 --> 00:03:00,440
by Simon Stone. Lawk has set his masterpiece amongst peasants in southern Spain and Simon

34
00:03:00,440 --> 00:03:05,280
has set his version in modern day London. Simon talks about his reasons for doing so

35
00:03:05,280 --> 00:03:10,920
in an exclusive short documentary coming up in a moment. Yerma is played without an interval

36
00:03:10,920 --> 00:03:15,960
and runs just over an hour and a half. It includes some strobe lighting and in this

37
00:03:15,960 --> 00:03:21,120
theatre it's going to be staged as almost magical. Although you're watching on screen

38
00:03:21,120 --> 00:03:26,200
we can assure you that seeing it live streamed and broadcast it will feel like you're just

39
00:03:26,200 --> 00:03:31,600
here with us. The performance will begin straight after this short film. I'm very excited as

40
00:03:31,600 --> 00:03:40,800
you can probably tell. Prepare for a gripping, intense and mesmerising experience.

41
00:03:40,800 --> 00:03:46,560
I'd been coming to the Young Vic when I happened to be in London quite a bit and I saw that

42
00:03:46,560 --> 00:03:53,440
the Young Vic consistently was the locus of significant cultural events and I thought

43
00:03:53,440 --> 00:03:58,160
to myself if one day I could ever work at that place then I would be an incredibly lucky

44
00:03:58,160 --> 00:04:08,240
human being. Do you know why I asked you to come and do a show with us? And if you don't

45
00:04:08,240 --> 00:04:12,720
I'm going to tell you. It happens to me from time to time that I see somebody's work and

46
00:04:12,720 --> 00:04:19,760
I go I want that. That is exactly what I think will speak to our work, our company, our audience,

47
00:04:19,760 --> 00:04:23,080
our part of the city. We were calling each other every couple of evenings trying to figure

48
00:04:23,080 --> 00:04:28,240
out what play I should do and then one night you just went I've had an idea and you said

49
00:04:28,240 --> 00:04:34,800
Yerma. I paused for a second and you started explaining why it would be a good idea and

50
00:04:34,800 --> 00:04:44,520
I think I interrupted you straight away and said that's a great idea. I think that's perfect.

51
00:04:44,520 --> 00:04:49,600
What about Lorca as a writer? What does he mean to you? I think Lorca is one of these

52
00:04:49,600 --> 00:04:55,180
writers you know there is a sense of urgency to the full length plays which is this might

53
00:04:55,180 --> 00:05:01,120
be the last thing I get to say and Yerma really has that feeling. And it almost was. It wasn't

54
00:05:01,120 --> 00:05:05,280
quite the last thing but it was almost the last thing and of course he didn't know it

55
00:05:05,280 --> 00:05:12,520
was coming. I mean he knew that fascism was growing in Spain and he would be like millions

56
00:05:12,520 --> 00:05:18,960
of other people under intense pressure but he didn't know he was going to die in 36.

57
00:05:18,960 --> 00:05:24,520
Federico Garcia Lorca's plays are interesting in many ways because he died very young. He

58
00:05:24,520 --> 00:05:31,040
died in 1936 a very early victim of the Spanish Civil War. He was shot by the Nationalists

59
00:05:31,040 --> 00:05:35,680
troops. One of the things that makes Lorca such a radical and important dramatist is

60
00:05:35,680 --> 00:05:41,200
that in most of his major plays women are at the center and here he's got a very active

61
00:05:41,200 --> 00:05:47,240
woman a woman who's longing for something that she feels society isn't giving her and

62
00:05:47,240 --> 00:05:52,840
in many ways a lot of what the play deals with is questioning conventional ideas of

63
00:05:52,840 --> 00:05:59,560
womanhood. Knowing what it was about and you know having read it at some point before I'd

64
00:05:59,560 --> 00:06:06,280
had my own experiences with struggling to try and have a child in a relationship. You

65
00:06:06,280 --> 00:06:11,840
read these plays or you read these myths or you read these novels and you store that material

66
00:06:11,840 --> 00:06:16,680
in you and at a certain point it finds the overlap with your own experience and then

67
00:06:16,680 --> 00:06:21,100
you kind of rediscover it and you trigger that for me. But there is something I think

68
00:06:21,100 --> 00:06:26,640
about the unexpected idea that when you go straight into the imagination and that's what

69
00:06:26,640 --> 00:06:35,000
seemed to happen with this. The word yerma of course means barren in Spanish so it's

70
00:06:35,000 --> 00:06:41,120
a deeply symbolic name. The physical landscape of the play becomes a way of thinking through

71
00:06:41,120 --> 00:06:47,040
the metaphorical journey that yerma undergoes. One of the things about the play that makes

72
00:06:47,040 --> 00:06:55,600
it quite difficult to stage is the fact that it has a very heightened lyrical poetic language

73
00:06:55,600 --> 00:07:01,840
that often in stage productions can feel highly artificial or labored and I think one of the

74
00:07:01,840 --> 00:07:08,920
strengths of Simon's adaptation is that it keeps the poetry of the original while finding

75
00:07:08,920 --> 00:07:19,880
a very immediate dynamic language for the play that makes it work in 2017. You've taken

76
00:07:19,880 --> 00:07:25,640
aspects of the play and used the metaphoric life of the play, the poetic life of the play

77
00:07:25,640 --> 00:07:33,260
and it emerges in what you've written in a way which is faithful to Lorca I would say

78
00:07:33,260 --> 00:07:37,400
but not in a direct way, in an indirect way. Do you think you've changed the play or is

79
00:07:37,400 --> 00:07:42,440
it the same play? I think it's the same myth. It's the same myth. It's not the same play

80
00:07:42,440 --> 00:07:48,080
because a play is what's written for the actors to perform in that season. I always look to

81
00:07:48,080 --> 00:07:54,900
somehow reflect the world that the audience has left when they walk into the theatre.

82
00:07:54,900 --> 00:08:00,800
Although they're coming into a space that is very much not the real world, it is a kind

83
00:08:00,800 --> 00:08:06,880
of kaleidoscope of references to the world that they have just exited and so that there's

84
00:08:06,880 --> 00:08:17,600
no possibility of creating some kind of nostalgic fantasy. The piece survives and is so overwhelmingly

85
00:08:17,600 --> 00:08:28,340
touching because they see these human beings bearing everything for the audience and that

86
00:08:28,340 --> 00:08:50,040
feels like such a privilege.
